corehq/apps/auditcare/couch_to_sql.py

```diff
@@ -93,7 +94,15 @@ def get_events_from_couch(start_key, end_key):
         if doc["doc_type"] == "NavigationEventAudit":
             nav_couch_ids.append(doc['_id'])
             kwargs.update(_pick(doc, ["headers", "status_code", "view", "view_kwargs"]))
-            path, _, params = doc.get("request_path", "").partition("?")
+            # Postgres does not play well with control characters in strings
+            # Some crafted URLs can contain these charachters, so replacing them with '' in request_path
+            # https://stackoverflow.com/a/14946355/3537212
+            request_path = re.sub(
+                r'[\x00-\x08\x0b\x0c\x0e-\x1f\x7f-\xff]',
+                '',
```

Comment:
Rather than replacing them with an empty string (removing them), how about replacing them with a representation of the character?
```suggestion
                lambda match: repr(match.group(0)),
```
(untested suggestion)
That way we can tell that there were strange characters in the URL.